data/en/mts_0001.json

```diff
@@ -0,0 +1,611 @@
+                                                "sales from catering facilities used by customers (for example, in-store cafes)",
+                                                "lottery sales and commission from lottery sales",
+                                                "sales of car accessories and motor vehicles",
+                                                "NHS receipts"
+                                            ]
+                                        }
+                                    ]
+                                },
+                                "description": "These are sales of goods to the general public (not businesses) for personal or household use.",
+                                "type": "General",
+                                "answers": [{
+                                    "id": "retail-sales-answer",
+                                    "mandatory": true,
+                                    "type": "Currency",
+                                    "label": "Retail sales including VAT",
+                                    "description": "",
+                                    "min_value": {
+                                        "value": 0,
+                                        "exclusive": false
+                                    },
+                                    "decimal_places": 0,
+                                    "currency": "GBP"
+                                }]
+                            }]
+                        }, {
+                            "type": "ConfirmationQuestion",
+                            "title": "Retail sales including VAT",
+                            "id": "confirm-zero-retail-sales-block",
+                            "questions": [{
+                                "type": "General",
+                                "answers": [{
+                                    "type": "Radio",
+                                    "id": "confirm-zero-retail-sales-answer",
+                                    "options": [{
+                                        "label": "Yes this is correct",
+                                        "value": "Yes"
+                                    }, {
+                                        "label": "No I need to change this",
+                                        "value": "No"
+                                    }],
+                                    "mandatory": true
+                                }],
+                                "id": "confirm-zero-retail-sales-question",
+                                "title": "For the period {{ format_conditional_date (answers['period-from'], metadata['ref_p_start_date'])}} to {{ format_conditional_date (answers['period-to'], metadata['ref_p_end_date'])}}, the value of retail sales was {{ format_currency(answers['retail-sales-answer'], 'GBP') }}, is this correct?"
```

Comment:
This filter is using wrong arguments. Hence its displaying wrong dates It should be used `period-from-answer` and `period-to-answer`

Reply:
Other places where we use the metadata date, are we supposed to be using the user inputted date?. Like here?

https://github.com/ONSdigital/eq-survey-runner/blob/1d9e309377f7c41f3c256526cf9749679fa671ed/data/en/mts_0001.json#L523

Reply:
Correct on both counts. Good spots.